### PR TITLE
Change to use loopback for #235

### PR DESCRIPTION
**Pull request by machawk1**
@ibnesayeed Please check this. PR'd so TravisCI can run tests.

### CODE REVIEW

**Comment by machawk1** [comment]
This is what you suggested in #235, @ibnesayeed , where you said, "Basically, you can just hard code app.run(host="0.0.0.0")".

Please suggest an alternative.

**Comment by machawk1** [comment]
I made this change, @ibnesayeed, just waiting on Travis again. The issue of customization is already described in #22 and #148 and is not yet enabled. Please check on the above ticket to let me know if it is satisfactory to close this PR and #235.

**Comment by ibnesayeed** [comment]
> This is what you suggested in #235, @ibnesayeed , where you said, "Basically, you can just hard code app.run(host="0.0.0.0")".
>
> Please suggest an alternative.

Yes, what I said is accurate. Basically, when you create a service (with the intent to listen on on network interfaces), you listen to `0.0.0.0`, but when you connect to a service, you can't use this non-resolvable address, instead used loopback address, localhost, or any other interface where the service is listening to.

File: ipwb/util.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 from ipfsapi.exceptions import ConnectionError
 
 
-IPFSAPI_IP = '127.0.0.1'
+IPFSAPI_IP = 'localhost'
```
Comment (ibnesayeed) [inline ipwb/util.py]: Using `localhost` here is equally as valid as `127.0.0.1`. This means, while this change is supposed to work as expected, this is not a **REQUIRED** change for this purpose.

Comment (machawk1) [reply ipwb/util.py]: localhost makes it consistent with IPWBREPLAY_IP, which is now a bit of a misnomer.

Thanks for the quick feedback.

**Pull request merged**